services/node/node-current.tester.js

```diff
@@ -0,0 +1,157 @@
+'use strict'
+
+const { expect } = require('chai')
+const { Range } = require('semver')
+const t = (module.exports = require('../tester').createServiceTester())
+const { mockPackageData, mockCurrentSha } = require('./testUtils/test-utils')
+
+function expectSemverRange(message) {
+  expect(() => new Range(message)).not.to.throw()
+}
+
+t.create('gets the node version of passport')
+  .get('/passport.json')
+  .expectBadge({ label: 'node' })
+  .afterJSON(json => {
+    expectSemverRange(json.message)
+  })
+
+t.create('engines satisfies current node version')
+  .get('/passport.json')
+  .intercept(
+    mockPackageData({
+      packageName: 'passport',
+      engines: '>=0.4.0',
+    })
+  )
+  .intercept(mockCurrentSha(13))
+  .expectBadge({ label: 'node', message: `>=0.4.0`, color: `brightgreen` })
+
+t.create('engines not satisfies current node version')
+  .get('/passport.json')
+  .intercept(
+    mockPackageData({
+      packageName: 'passport',
+      engines: '12',
+    })
+  )
+  .intercept(mockCurrentSha(13))
+  .expectBadge({ label: 'node', message: `12`, color: `yellow` })
+
+t.create('gets the node version of @stdlib/stdlib')
+  .get('/@stdlib/stdlib.json')
+  .expectBadge({ label: 'node' })
+  .afterJSON(json => {
+    expectSemverRange(json.message)
+  })
+
+t.create('engines satisfies current node version - scoped')
+  .get('/@stdlib/stdlib.json')
+  .intercept(
+    mockPackageData({
+      packageName: 'stdlib',
+      engines: '>=0.4.0',
+      scope: '@stdlib',
+      tag: '',
+      registry: '',
+    })
+  )
+  .intercept(mockCurrentSha(13))
+  .expectBadge({ label: 'node', message: `>=0.4.0`, color: `brightgreen` })
+
+t.create('engines not satisfies current node version - scoped')
+  .get('/@stdlib/stdlib.json')
+  .intercept(
+    mockPackageData({
+      packageName: 'stdlib',
+      engines: '12',
+      scope: '@stdlib',
+      tag: '',
+      registry: '',
+    })
+  )
+  .intercept(mockCurrentSha(13))
+  .expectBadge({ label: 'node', message: `12`, color: `yellow` })
+
+t.create("gets the tagged release's node version version of ionic")
```

Comment:
I'm a little surprised that prettier didn't flag the double quotes here (and below) :thinking: